modules/aws/iam.go

```diff
@@ -1,14 +1,60 @@
 package aws
 
 import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"net/url"
 	"testing"
 	"time"
 
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/service/iam"
 	"github.com/gruntwork-io/terratest/modules/logger"
+	"github.com/stretchr/testify/assert"
 )
 
+// NoPolicyError is used to alert that a valid policy was found from an ARN but a document doesn't exist against it.
+type NoPolicyError struct {
+	error
+}
+
+func (e *NoPolicyError) Error() string {
+	return "the policy exists but no policy document was returned"
+}
+
+// AssertJsonEqual takes two json strings and compares them
+func AssertJsonEqual(t *testing.T, correct string, comparison string) {
+	err := AssertJsonEqualE(t, correct, comparison)
+	if err != nil {
+		t.Fatal(err)
+	}
+	return
+}
+
+// AssertJsonEqualE takes two json strings and compares them, it returns an error if they're not identical
+func AssertJsonEqualE(t *testing.T, correct string, comparison string) error {
+	dst := new(bytes.Buffer)
+
+	src := []byte(correct)
+
+	err := json.Compact(dst, src)
+	if err != nil {
+		return err
+	}
+
+	equal := assert.JSONEq(
```

Comment:
This will fail the test it they are not equal, which violates the contract of the methods that end in `E`. You'll need to do the JSON comparison using some other approach.